telemetry/data_feature_usage.go

```diff
@@ -61,7 +62,7 @@ func getFeatureUsage(ctx sessionctx.Context) (*featureUsage, error) {
 	var err error
 	usage.NewClusterIndex, usage.ClusterIndex, err = getClusterIndexUsageInfo(ctx)
 	if err != nil {
-		logutil.BgLogger().Info(err.Error())
+		logutil.BgLogger().Info("Failed to get feature usage", zap.Error(err))
```

Comment:
How about make it more clear, for example something like: 'Failed to get cluster index usage'?

Reply:
Just realized this is a cherry-pick. Please ignore this comment